models/cow_protocol/ethereum/cow_protocol_ethereum_trades.sql

```diff
@@ -218,7 +218,8 @@ valued_trades as (
            limit_sell_amount,
            limit_buy_amount,
            valid_to,
-           flags
+           flags,
+           case when (flags % 2) = 0 then 'SELL' else 'BUY' end as order_type
```

Comment:
Isn't flags a string field? I wonder how it can be evaluated mod 2.

Reply:
That's a good point. We should make sure to have the correct column type here.